book/extend-admin.rst

```diff
@@ -262,7 +265,7 @@ as well, then you should be able to see these actions when using the ``debug:rou
 
 .. code-block:: bash
 
-    $ bin/adminconsole debug:router | grep event
+    $ bin/adminconsole debug:router
```

Comment:
`| grep` is unix specific and I would avoid unix specific commands in the docs

Reply:
But in that case it absolutely makes sense in my opinion. Otherwise you have to skim through over 200 routes... But we could also add two different commands, to also show how this is done on windows.